ardere/aws.py

```diff
@@ -107,6 +114,10 @@ def s3_ready_file(self):
             key="{}.ready".format(self._plan_uuid)
         )
 
+    @property
+    def influx_db_name(self):
+        return "run-{}".format(self.plan_uuid)
```

Comment:
It seems weird to call it 'loads' when its not called loads-broker anymore. Since we don't really have an archive of them anywhere yet, I doubt it matters much. Ideally loads-broker will be dropped once ardere is up to it.